src/utils/urlEncoding.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Decodes symbol which is URL encoded.
+ * Trims trailing and leading spaces.
+ *
+ * @param symbol URL encoded symbol
+ */
+export function decodeSymbol(symbol: string): string {
+  return decodeURIComponent(symbol).trim()
+}
+
+const dashRegex = new RegExp(/-/, 'g')
```

Comment:
This is a minor annoyance, maybe even personal minor annoyance, but when source of RegExp isn't dynamic, don't create it through a constructor. It copies unnecessarily
Also see gnosis/dex-react#933
```suggestion
const dashRegex = /-/g
```